src/handlers/int-test-support/helpers/lambdaHelper.ts

```diff
@@ -32,3 +32,22 @@ export const sendLambdaCommand = async <THelper extends IntTestHelpers>(
     return "Error";
   }
 };
+
+export const invokeLambda = async (
```

Comment:
I don't understand this change, can you help me out?

Reply:
Is the idea to use `sendLambdaCommand` and `invokeLambda` in different circumstances?

Reply:
sendLamdaCommand is called when the tests run in pipeline I was thinking to reuse this function to trigger the transaction events via filterLambda but the sendLambda has different payload and parameters so added this invokeLambda function which takes only function and payload

Reply:
>Is the idea to use sendLambdaCommand and invokeLambda in different circumstances? 

⬇️ Below summarises a convo I had with @NKannan14 and some playing around with the code I did to get an understanding ⬇️ 

Yes. `invokeLambda` is used to send an given payload to a given lambda whereas `sendLambdaCommand` sends a given payload to a specific lambda - namely the int-test-support-function. `invokeLambda` facilitates end to end testing by allowing us to kick off the filter lambda directly. We need to invoke the filter directly because [REDACTED] team have the SQS that we subscribe to in their env, so we can't pop things on it ourselves.